Source/WebKit/Shared/Cocoa/WebCoreArgumentCodersCocoa.serialization.in

```diff
@@ -22,6 +22,44 @@
 
 headers: "ArgumentCodersCocoa.h"
 
+#if USE(CORE_TEXT)
+class WebCore::FontPlatformData {
+    float size();
+    WebCore::FontOrientation orientation();
+    WebCore::FontWidthVariant widthVariant();
+    WebCore::TextRenderingMode textRenderingMode();
+    bool syntheticBold();
+    bool syntheticOblique();
+    WebCore::FontPlatformData::PlatformDataVariant platformSerializationData();
+}
+
+header: <WebCore/FontPlatformData.h>
+[CustomHeader] struct WebCore::FontPlatformDataSerializationWithCreationData {
+    Vector<uint8_t> fontFaceData;
+    RetainPtr<CFDictionaryRef> attributes;
```

Comment:
This used to always be encoded, and now it is encoded only if the PlatformDataVariant is FontPlatformDataSerializationWithCreationData.  Is that an intentional change?

Reply:
`RetainPtr<CFDictionaryRef> attributes` is in both structs.